modules/test/playwright/tests/notifications-web/pages/NotificationsPage.ts

```diff
@@ -22,8 +22,13 @@ export class NotificationsPage {
 		});
 	}
 
-	async goto() {
-		await this.page.getByLabel('Test Test User Profile').click();
+	async goto(userName?: string) {
+		if (userName) {
+			await this.page.getByLabel(` ${userName} User Profile`).click();
+		}
+		else {
+			await this.page.getByLabel('Test Test User Profile').click();
+		}
```

Comment:
I think this can be simplified having the default parameter
```
async goto(userName = 'Test Test') {
		await this.page.getByLabel(` ${userName} User Profile`).click();
```
because at this point you will always have a userName and you don't have to do the check